qa-core/src/config/public-api/TestConfiguration/accountsAPIClient.ts

```diff
@@ -0,0 +1,65 @@
+import env from "../../../environment"
+import fetch, { HeadersInit } from "node-fetch"
+
+type APIMethod = "GET" | "POST" | "PUT" | "PATCH" | "DELETE"
+
+interface ApiOptions {
+  method?: APIMethod
+  body?: object
+  headers?: HeadersInit | undefined
+}
+
+class AccountsAPIClient {
```

Comment:
Is it necessary to create a completely new API client here? It seems quite similar to the one that lives in `PublicAPIClient`. NAB for this PR, but we should think about combining them in some way.

Reply:
@shogunpurple Yes because the URL they are calling is different. I have plans to refactor the API tests because there is code repetition in both internal and public API configurations and different services (with different urls) require different API client classes to be defined, but they share all the same functionality, including error handling. So it seems we need a base API client class.